static/js/util/ProfileFormFields.js

```diff
@@ -64,13 +66,41 @@ export default class ProfileFormFields extends React.Component {
     }));
   }
 
+  stepBack = () => {
+    this.context.router.push(this.prevUrl);
+  };
+
   saveAndContinue = () => {
-    let lastTab = this.nextUrl === "/dashboard";
-    saveAndContinue.call(this, lastTab).then(() => {
+    saveProfileStep.call(this, this.isLastTab).then(() => {
       this.context.router.push(this.nextUrl);
     });
   };
 
+  progressControls = (cellWidth=10) => {
+    let prevButton, nextButton;
+    if(this.prevUrl) {
+      prevButton = <Button
+        raised
+        className="progress-button previous"
+        onClick={this.stepBack}>
+        <span>Previous</span>
+      </Button>;
+    }
+    if(this.nextUrl) {
+      nextButton = <Button
+        raised
+        colored
+        className="progress-button next"
+        onClick={this.saveAndContinue}>
+        <span>{this.isLastTab ? "I'm Done!" : "Save and Continue"}</span>
+      </Button>;
+    }
+    return <Cell col={cellWidth}>
+        {prevButton}
+        {nextButton}
+      </Cell>;
```

Comment:
i thought about that, but didn't go for it initially b/c it's just adding an extraneous html element. looking at it with fresh eyes, i think that's preferable to what is here currently. i think an even cleaner solution might be using an actual react component rather than a simple function. something like...

```javascript
return <Cell {...this.props}>
  {prevButton}
  {nextButton}
</Cell>;

// then in the parent component
<Cell col={2} />
<ProgressControlCell col="10" />
<Cell col={2} />
```

not perfect, but put that forward b/c i've been wondering why we don't do this more in general. seems like a lot of our components are simple function calls. i can't say for sure whether or not we're missing out on important functionality, but i think we definitely lose some semantic cleanliness when some components in a `render` method are appear as `<SomeComponent />` and others appear as `{this.someComponent()}`. 

anyway, i'll change this to your suggestion